radix-engine-stores/src/rocks_db_with_merkle_tree/mod.rs

```diff
@@ -10,7 +12,9 @@ use rocksdb::{
     SingleThreaded, WriteBatch, DB,
 };
 use sbor::rust::prelude::*;
+use sbor::HasLatestVersion;
```

Comment:
Even easier is `use sbor::prelude::*`, which also subsumes the previous line.

Reply:
whoops,
```
error[E0599]: no method named `into_latest` found for enum `VersionedTreeNode` in the current scope
   --> /home/runner/work/radixdlt-scrypto/radixdlt-scrypto/radix-engine-stores/src/rocks_db_with_merkle_tree/mod.rs:223:40
    |
223 |               .map(|versioned| versioned.into_latest())
    |                                          ^^^^^^^^^^^ method not found in `VersionedTreeNode`
```
(https://github.com/radixdlt/radixdlt-scrypto/actions/runs/6161819412/job/16721912689?pr=1514)

Although I thought it compiled locally in intellij o_O

Reply:
Ohh IntoLatest should definitely be in the `sbor` prelude. If it's missing, it should be added :)

Reply:
ok, fixed 👍